dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -136,6 +141,12 @@ public abstract class AbstractJdbcEventAnalyticsManager {
 
   private static final String COL_VALUE = "value";
 
+  private static final String COL_PERIOD = "pe";
+
+  private static final String COL_ORGANIZATION = "ou";
```

Comment:
I'm pretty sure we already have these defined somewhere else. Does it still makes sense to duplicate here ?

Reply:
done